org.gnome.GHex.json

```diff
@@ -5,23 +5,31 @@
     "sdk": "org.gnome.Sdk",
     "command": "ghex",
     "finish-args": [
-        /* X11 + XShm */
-        "--share=ipc", "--socket=x11",
-        /* Wayland */
+        "--device=dri",
+        "--filesystem=host",
+        "--filesystem=xdg-run/gvfsd",
+        "--share=ipc",
+        "--socket=fallback-x11",
         "--socket=wayland",
-        /* Filesystem */
-        "--filesystem=host"
+        "--socket=x11",
```

Comment:
why socket=x11 was kept if you added fallback-x11?

Reply:
It doesn't hurt, fallback-x11 will still be respected, and it keeps compatibility with older Flatpak releases, but I guess we don't care about flatpak<1.0?